src/sagemaker_containers/env.py

```diff
@@ -228,6 +228,7 @@ def __init__(self):
         module_dir = os.environ.get(SUBMIT_DIR_ENV, None)
         enable_metrics = util.strtobool(os.environ.get(ENABLE_METRICS_ENV, 'false')) == 1
         log_level = os.environ.get(LOG_LEVEL_ENV, logging.INFO)
+        framework_module = os.environ.get(FRAMEWORK_MODULE_ENV, ' ')
```

Comment:
Should we fail fast here? What happens if the env vars is not passed? Or should we add a default value.

Reply:
Nvm, I just noticed your todo